src/helpers/credential.rs

```diff
@@ -269,6 +269,50 @@ impl<'a> Credential<'a> {
         Ok(())
     }
 
+    #[cfg(feature = "plugin-did-sidetree")]
+    pub async fn revoke_credential(
```

Comment:
Can we add a small function header documentation?

Reply:
done